compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -110,6 +110,12 @@ int entry(int argc, char **argv)
       .default_value(false)
       .help("This will fuse BatchNorm operators of pre-activations to Convolution operator");
 
+  arser.add_argument("--remove_redundant_transpose")
+      .nargs(0)
+      .required(false)
+      .default_value(false)
```

Comment:
How about setting default_value as `true` for this?

Reply:
then need to add to `--all` too

Reply:
both gets same result. Just adding it to `--all` section is enough.

Reply:
Okay, I will add on `--all` section.